lib/stat_tracker.rb

```diff
@@ -1,24 +1,34 @@
+require 'csv'
+require './lib/game_teams_collection'
+require './lib/game_collection'
+require './lib/team_collection'
+
 class StatTracker
+
+  def self.from_csv(locations)
+    raw_data = {}
+    raw_data[:game_data] = CSV.read(locations[:games], headers: true, header_converters: :symbol)
+    raw_data[:team_data] = CSV.read(locations[:teams], headers: true, header_converters: :symbol)
+    raw_data[:game_teams_data] = CSV.read(locations[:game_teams], headers: true, header_converters: :symbol)
+
+    StatTracker.new(raw_data)
+  end
+
   attr_reader :game_collection, :team_collection, :gtc
-  def initialize(locations)
-    @game_file_path = locations[:games]
-    @teams_file_path = locations[:teams]
-    @game_teams_file_path = locations[:game_teams]
+  def initialize(raw_data)
+    @game_data = raw_data[:game_data]
+    @team_data = raw_data[:team_data]
+    @game_teams_data = raw_data[:game_teams_data]
     @gtc = nil
     @game_collection = nil
     @team_collection = nil
+    construct_collections
   end
 
-  def load_game_team_data
-    @gtc = GameTeamsCollection.new(@game_teams_file_path)
-  end
-
-  def load_game_data
-    @game_collection = GameCollection.new(@game_file_path)
-  end
-
-  def load_team_data
-    @team_collection = TeamCollection.new(@teams_file_path)
+  def construct_collections
```

Comment:
Love that all this got put into one line, as I thought about it more, there was no need for each to have its own methods.